packages/snippet-bot/src/snippet-bot.ts

```diff
@@ -450,7 +451,61 @@ To update this comment, add \`${REFRESH_LABEL}\` label.
   );
 }
 
+// This should synced with the body of the comment. When we move
+// `onRefreshCheckbox` to gcf-utils, we can have the single source and
+// automatically add the checkbox in `addOrUpdateIssueComment`.
+const REFRESH_STRING = '- [x] Refresh this comment';
+
+function onRefreshCheckbox(
+  app: Application,
+  callback: (context: Context) => Promise<void>
+): void {
+  app.on('issue_comment.edited', async context => {
```

Comment:
API question: I kind of like having all `app.on` calls in the main application function. That way, you can easily see what event types lead to what code being run. But, if we call `app.run` here (or in gcf-utils), this `issue_comment.edited` event is hidden. Perhaps we should lift the `app.on` call to the caller of `onRefreshCheckbox`?

This would increase the verbosity of the API, but I believe it will make it easier to understand and debug.

Reply:
Sure, in gcf-utils, maybe we can only provide a function for creating commentMark `getCommentMark(instanceId: string)`, so that the user handler can determine if the comment is made by the bot.

The logic after that might defer bot by bot (one bot wants to "refresh the comment", but another bot might want to "rebase", or "disable the feature", or whatever).

I'll update this PR accordingly.

Reply:
Greatly simplified the API. PTAL